### PR TITLE
Adding support for chunked chats

### PR DESCRIPTION
**Pull request by danieldoglas**
### Details
Adding new events for pusher with chunked push notifications.

### Fixed Issues
$ https://github.com/Expensify/App/issues/3797

### Tests
1. Run new.E locally pointing to production Web-E
2. Open two different accounts and start a chat with each other
3. Write (or paste) any messages that are bigger than 10240 characters
4. Verify that the message will appear for the receiver without needing to reload the chat.

### QA Steps
1. Run new.E locally pointing to production Web-E
2. Open two different accounts and start a chat with each other
3. Write (or paste) any messages that are bigger than 10240 characters
4. Verify that the message will appear for the receiver without needing to reload the chat.

### Tested On

- [X] Web
- [X] Mobile Web
- [X] Desktop
- [X] iOS
- [X] Android

#### Web/Desktop/iOS/Android

https://user-images.githubusercontent.com/5818999/145457568-2c01b9b4-7212-412a-aeae-6c755fd5047e.mov

### CODE REVIEW

**Comment by danieldoglas** [comment]
Fixed JS Lints.

File: src/libs/actions/Report.js [hunk 1]
```diff
@@ -735,6 +753,26 @@ function subscribeToUserEvents() {
...
 
+    // Live-update a report's actions when an 'edit comment chunk' event is received.
+    Pusher.subscribe(pusherChannelName, Pusher.TYPE.REPORT_COMMENT_EDIT_CHUNK, (pushJSON) => {
+        Log.info(
+            `[Report] Handled ${Pusher.TYPE.REPORT_COMMENT_EDIT_CHUNK} event sent by Pusher`, false, {
+                reportActionID: pushJSON.reportActionID,
+            },
+        );
+        updateReportActionMessage(pushJSON.reportID, pushJSON.sequenceNumber, pushJSON.message);
+    }, false,
```
Comment (yuwenmemon) [inline src/libs/actions/Report.js:764]: This is `isChunked`, right? Should this value actually be `true`?

**Review by yuwenmemon — COMMENTED** [review]
Looks good! But one question for ya

**Comment by danieldoglas** [comment]
Fixed flag for comment edit.

**Review by yuwenmemon — APPROVED** [review]

**Review by parasharrajat — COMMENTED** [review]
Do you think we should also clean the duplicate call at L149 in `pusher.js`.

**Pull request merged**